games/minetest-devel/Portfile

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               cmake 1.1
+
+compiler.cxx_standard   2011
+compiler.thread_local_storage   yes
+
+name                    minetest-devel
+github.setup            minetest minetest 5.6.1
+revision                0
+
+# the game version could be different to the minetest version
+set game_version        5.6.1
+
+# to add on more files to a github portgroup download
+# have to cache the preset distfiles from the github PG
+set main_distfile       ${distfiles}
+
+# then add another distfile, with a direct URL as can't use the github PG again
+set game_distfile       ${game_version}${extract.suffix}
+set game_mastersite     https://github.com/minetest/minetest_game/archive/
+
+distfiles               ${main_distfile}:main \
+                        ${game_distfile}:game
+
+master_sites            ${github.master_sites}:main \
+                        ${game_mastersite}:game
+
+checksums               ${main_distfile} \
+                        rmd160  7b32331e0007f29003107592522d1c906935e44d \
+                        sha256  425d8e9363212dcb5bc7986dd12460c9c3784aca22eb1b90dad6a7fc2c6ef546 \
+                        size    9925064 \
+                        ${game_distfile} \
+                        rmd160  73d35423b856d141cebfa16396fe5dcf834dcf89 \
+                        sha256  5dc857003d24bb489f126865fcd6bf0d9c0cb146ca4c1c733570699d15abd0e3 \
+                        size    2590582
+
+# rename directory - from github portgroup
+post-extract {
+    if {[file exists [glob -nocomplain ${workpath}/${github.author}-${github.project}-*]] && \
+        [file isdirectory [glob -nocomplain ${workpath}/${github.author}-${github.project}-*]]} {
+        move [glob ${workpath}/${github.author}-${github.project}-*] ${workpath}/${distname}
+    }
+}
+
+license                 EUPL-1.2
```

Comment:
this refers to the license of the package. Whatever changes/addition to Portfiles you provide will be under MacPort's license, which is BSD as far as I know. 

Looking at upstream this should be LG-2.1

Reply:
THX for the advice. I agree, there would be **no need for the EUPL-1.2 in a.m. Portfile**.

_NOTE: Furthermore, my a.m. note on the EUPL-1.2 in this PR fulfills my legal requirements as maintainer and/or contributor of PRs from the EU quite sufficiently. I don't seek to make your life (or life at MacPorts in general) more complicated. You kindly explained the already nested licenses of this port quite acceptably._

In conclusion the Portfile shall continue with "LGPL-1.2+" as proposed. However, **`LGPL` is the more correct indicator** if I am correct. One should always follow approved OSI indicators, I presume

See: <https://opensource.org/licenses/lgpl-license>

I would deduce that **`LGPL` is the most applicable indicator**. 

Please note an _unofficial_ indicator like "LGPL-1.2+" would imply "LGPL-3+" and newer versions thus "LGPL" which implies just that. Would you agree?

Reply:
NB - One could argue my former indicator could have been "EUPL" which would be correct. However, this would not make "EUPL-1.2" incorrect as laid down by the EUPL-1.2 text and OSI approval.